requirements.txt

```diff
@@ -8,5 +8,4 @@ unittest2
 coverage
 pygments
 pymongo
-docker
-psutil
\ No newline at end of file
+docker
```

Comment:
Do we really want docker in the requirements?

Reply:
Yes we do. That's how we control the docker instances.